src/index.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+
+// const axios = require('axios');
+const state = {
+  city: '',
+};
+
+function incrementButton() {
+  const element = document.getElementById('counter');
+  let value = element.innerHTML;
+
+  value++;
+
+  temperatureColor(value);
+  landscapeIcons(value);
+
+  document.getElementById('counter').innerHTML = value;
+}
+
+function decreaseButton() {
+  const element = document.getElementById('counter');
+  let value = element.innerHTML;
+
+  value--;
+
+  temperatureColor(value);
+  landscapeIcons(value);
+
+  document.getElementById('counter').innerHTML = value;
+}
+
+function temperatureColor(temp) {
+  let element = document.getElementById('counter');
+  if (temp >= 80) {
+    element.className = 'red';
+  } else if (temp >= 70 && temp <= 79) {
+    element.className = 'orange';
+  } else if (temp >= 60 && temp <= 69) {
+    element.className = 'yellow';
+  } else if (temp >= 50 && temp <= 59) {
+    element.className = 'green';
+  } else if (temp <= 49) {
+    element.className = 'teal';
+  }
+}
+
+function landscapeIcons(temp) {
+  let emoji = document.getElementById('emojis');
+  if (temp >= 80) {
+    emoji.innerText = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (temp >= 70 && temp <= 79) {
+    emoji.innerText = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (temp >= 60 && temp <= 69) {
+    emoji.innerText = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (temp <= 59) {
+    emoji.innerText = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
+}
+
+function skySelector(dropDownEvent) {
+  let dropdownEmojis = document.getElementById('dropdown-emojis');
+  let dropdownValue = dropDownEvent.currentTarget.value;
+  let skyEmojiValue = dropDownEvent.target.selectedOptions[0].value;
+  if (skyEmojiValue === 'sunny') {
+    dropdownEmojis.innerText = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️';
+  } else if (skyEmojiValue === 'cloudy') {
+    dropdownEmojis.innerText = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+  } else if (skyEmojiValue === 'rainy') {
+    dropdownEmojis.innerText = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+  } else if (skyEmojiValue === 'snowy') {
+    dropdownEmojis.innerText = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+  }
+}
+
+const cityNameInput = (e) => {
+  const log = document.getElementById('City-Name-Top');
+  state.city = document.querySelector('#city-input').value;
+  log.textContent = state.city;
+  console.log(e.data);
+};
```

Comment:
We should use `getElementById` on line 77 instead if we're just getting an element through its id. We could also do `e.target.value` here, which would be more flexible.

`log` is a bit confusing of a variable name, can you think of a clearer one?

Remember to delete `console.log`s once they're no longer needed.